### PR TITLE
Notes interest in security talks for Pittsburgh 2015

### PR DESCRIPTION
**Pull request by colindean**
Need some agreement from @sethvargo @cmluciano @carsomyr @meatballhat @littleidea @j41manning, but this is the same language  we discussed on Friday.

### CODE REVIEW

**Comment by cmluciano** [comment]
Wondering if it should be something more along the lines of "We are especially interested in computer security topics and how they relate to devops". "devops security" seems kind of confusing to me.

**Comment by colindean** [comment]
Agreed. Fixing.

File: site/content/events/2015-pittsburgh/propose/index.html [hunk 1]
```diff
@@ -18,6 +18,9 @@
 This year, we are accepting all full-length (30-45 minute) proposals through our blinder system. 
 Talk proposals are accepted until <%= render(:partial => "/#{@eventhome}/_cfp_date_end") %>.
 
+We are especially interested in talks about computer ssecurity and social engineering as they relate to DevOps,
```
Comment (sethvargo) [inline site/content/events/2015-pittsburgh/propose/index.html]: I would say "computer, information, and human security and social engineering as they related to DevOps"

Comment (colindean) [inline site/content/events/2015-pittsburgh/propose/index.html]: Fixed.

**Comment by colindean** [comment]
I think this is ready to go, pending clearance from @sethvargo

**Comment by cmluciano** [comment]
This reads like a run-on sentence. I think human security and social engineering could be combined and the last part could be split into another sentence.

"We are especially interested in talks regarding computer & information security as well as social engineering. While we are looking for these specifically, all topics related to DevOps will be considered."

**Comment by colindean** [comment]
@cmluciano: Fixed. I kept the “as they relate” part in there to encourage that there be a devops focus to the proposals.

**Comment by sethvargo** [comment]
Seems okay to me

**Pull request merged**